### ISSUE TITLE
Delete Dialog Readability

### ISSUE THREAD

**Issue by Coros** [jamf/JamfSync#45]
When using a dark mode UI and attempting to delete a package through Jamf Sync, the dialog that asks to delete the files or package record uses red text.  That text is hard to read.  Please adjust the text color when in dark mode.

I go a little cross eyed every time I read it.

<img width="281" alt="Screenshot 2024-11-18 at 8 18 51 AM" src="https://github.com/user-attachments/assets/42e4ff03-6346-40c7-b81e-2836485230cf">

**Comment by HarryStrandJamf** [comment]
I agree with you that the colors aren't great, but this particular view is provided by apple which is choosing the colors. I could make something custom, but there are two reasons not to. One is that the time would be better spent elsewhere. The other is that wouldn't be as robust as Apple changes things in the future (including possibly fixing this color issue). So I'm closing this one.
